test/integration/sessions_test.rb

```diff
@@ -0,0 +1,10 @@
+require "integration_test_helper"
+
+class SessionsTest < ActionDispatch::IntegrationTest
+  context "when logged in" do
+    should "redirect the user to the account manager URL if they visit /sign-up" do
+      get "/sign-in", headers: { "HTTP_GOVUK_ACCOUNT_SESSION" => "placeholder" }
```

Comment:
Can we use the external name of the header, `GOVUK-Account-Session` here?  The point of this test is to check that Rails is actually renaming the header internally.